docs/zkasm/test_summary.csv

```diff
@@ -1,7 +1,9 @@
 Suite path,Passing count,Total count,Total cycles
+<<<<<<< HEAD,,,
```

Comment:
Looks like a merge conflict

Reply:
Maybe we should change our CI such that it will check if summary formatted properly?